iocage_lib/ioc_stop.py

```diff
@@ -223,9 +223,14 @@ def __stop_jail__(self):
             for nic in self.nics.split(","):
                 nic = nic.split(":")[0]
                 try:
-                    iocage_lib.ioc_common.checkoutput(
-                        ["ifconfig", f"{nic}.{self.jid}", "destroy"],
-                        stderr=su.STDOUT)
+                    if self.conf['netgraph'] == 'on':
```

Comment:
We can just test for the value now with the recent bool pr being merged
```suggestion
                    if self.conf['netgraph']:
```